_config.yml

```diff
@@ -1,4 +1,4 @@
-highlighter: rouge
```

Comment:
Ok, but why not using the new default highlighter ? 
They are not fully compatible ?